File: lp2/PokemonV-Java/src/lp2/Potion.java
package lp2;

public class Potion extends Item {

    private int hp;

    public Potion(int hp, String name, String description) {
        super(name, description);
        this.hp = hp;
    }

    public int getHp() {
        return hp;
    }

    public void setHp(int hp) {
        this.hp = hp;
    }

    public void pour() {

    }
}

File: lp2/PokemonV_Server/Entidades/Potion.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Entidades
{
    public class Potion : Item
    {
        public Potion(string name, string descrip, int points)
            : base(name, descrip)
        {
            this.healthPoints = points;
        }

        public void pour()
        {

        }

        protected int healthPoints;
        public int HealthPoints
        {
            get { return healthPoints; }
            set { healthPoints = value; }
        }
    }
}
